src/admin/content/content.gql.ts

```diff
@@ -40,6 +42,16 @@ export const GET_CONTENT = gql`
 	}
 `;
 
+export const GET_CONTENT_COUNT = gql`
+	{
+		app_content_aggregate {
+			aggregate {
+				count
+			}
+		}
+	}
+`;
```

Comment:
dit kan je combineren in 1 query:
```
export const GET_CONTENT = gql`
	query getContent(
		$where: app_content_bool_exp
		$offset: Int = 0
		$limit: Int = 10
		$order: [app_content_order_by!] = {}
	) {
		app_content(where: $where, limit: $limit, offset: $offset, order_by: $order) {
			content_type
			created_at
			depublish_at
			description
			id
			is_protected
			is_public
			is_published
			path
			profile {
				user: usersByuserId {
					first_name
					last_name
					role {
						id
						label
					}
				}
			}
			publish_at
			title
			updated_at
		}
		app_content_aggregate {
			aggregate {
				count
			}
		}
	}
`;
```

Reply:
Nice, wist ik niet 👍

Reply:
en als je 2 keer uit dezelfde tabel wil selecteren kan je dat oplossen door een alies ervoor te zetten:
```
{
  firstResult: app_content(where ....)
  secondResult: app_content(where ....)
}
```

Reply:
Fixed